edx_django_utils/data_generation/tests/test_management.py

```diff
@@ -0,0 +1,501 @@
+"""
+Test management commands and related functions.
+"""
+
+import mock
+from django.core.management import get_commands, load_command_class
+from django.core.management.base import CommandError, SystemCheckError
+from django.test import TestCase
+from pytest import mark
+
+from edx_django_utils.data_generation.management.commands.manufacture_data import Command, Node
+# pylint: disable=unused-import
```

Comment:
Small NIT- do you think it's worth a comment explaining disabling the linter check?

Reply:
Added!